### DISCUSSION TITLE
I can't download the app v6.4.5

### DISCUSSION THREAD

**Discussion by G4bb1** [Sketchware-Pro/Sketchware-Pro#673] [Q&A] [answered]
I can't download the app, a notification appears that it requires authorizes.

**Comment by j-px** [comment, marked as answer]
You might have to log in with a GitHub account to download releases. We can't fix that unfortunately.

**Comment by G4bb1** [comment]
I managed to download it, thanks for replying. But, I installed v6.4.0-rc04, v6.4.0-rc05 has a virus.
Now the problem is that I can't edit any project. I can only create new projects. Does anyone have any idea what the problem is?
My cell phone is a galaxy A11, android version: 12.

**Reply by j-px** [reply]
And why can't you edit new projects? Is the app crashing or what?

Also, v6.4.0-rc05 being a virus is a false-positive detection. Really not sure why. In my opinion, only when Play Protect steps in the app is malware.

**Reply by G4bb1** [reply]
The project opens normally, but when I click on the buttons nothing happens. I can delete the project, change its colors, add the logo, generate the project apk, install and open normally. And even the sketchware settings I can change, I set it to save automatically and whenever I exit the project a notification appears that the changes were saved.

**Reply by j-px** [reply]
What's your device?

**Reply by G4bb1** [reply]
Samsung galaxy A11

**Reply by j-px** [reply]
Could you maybe show a screen recording of that?

**Comment by G4bb1** [comment]
thank you so much for being willing to help. Now I can edit my projects. I don't know what happened, because I uninstalled the app, restarted the phone and messed with the settings. But now sketchware works perfectly on my mobile phone. Thanks again and I wish you all the best.
